src/repository/tag-repository.js

```diff
@@ -0,0 +1,34 @@
+import { PrismaClient } from '@prisma/client';
+
+const prisma = new PrismaClient();
+
+class TagRepogitory {
+  async getTags(where, order, skip, take) {
+    return await prisma.tag.findMany({
+      where,
+      orderBy: { created_at: order },
+      skip,
+      take,
+      select: {
+        id: true,
+        name: true,
+        created_at: true,
+        updated_at: true,
+      },
+    });
+  }
+
```

Comment:
```suggestion
  async tagsCount(where) {
    return prisma.tag.count({where});
  }
```
태그의 총 갯수를 반환하는 리포지토리 코드입니다.